fe/fe-core/src/main/java/org/apache/doris/datasource/iceberg/IcebergMetadataCache.java

```diff
@@ -177,29 +170,6 @@ public void invalidateDbCache(long catalogId, String dbName) {
                 });
     }
 
-    private Catalog createIcebergHiveCatalog(String uri, Map<String, String> hdfsConf, Map<String, String> props) {
-        // set hdfs configure
-        Configuration conf = DFSFileSystem.getHdfsConf(
-                hdfsConf.getOrDefault(DFSFileSystem.PROP_ALLOW_FALLBACK_TO_SIMPLE_AUTH, "").isEmpty());
-        for (Map.Entry<String, String> entry : hdfsConf.entrySet()) {
-            conf.set(entry.getKey(), entry.getValue());
-        }
-        HiveCatalog hiveCatalog = new HiveCatalog();
-        hiveCatalog.setConf(conf);
-
-        if (props.containsKey(HMSExternalCatalog.BIND_BROKER_NAME)) {
```

Comment:
This part is missing?

Reply:
I kept this part and removed the `else` part. As it seemed to me that the two were duplicates. The `else` part just initialize hivecatalog use `uri` propertie.